src/AnnotationDialog.js

```diff
@@ -176,6 +177,10 @@ class AnnotationDialog extends EventEmitter {
         this.hideMobileDialog = this.hideMobileDialog.bind(this);
         dialogCloseButtonEl.addEventListener('click', this.hideMobileDialog);
 
+        if (this.hasTouch) {
+            dialogCloseButtonEl.addEventListener('touchstart', this.hideMobileDialog);
```

Comment:
Can you move this to `bindDOMListeners`?

Reply:
Same with the above listener for `click`

Reply:
👍